test/template-create/expected-stdout

```diff
@@ -5,6 +5,7 @@ Created readme file "README.md".
 Plan for "pull" operation:
   + C extractor/ex-generic-v2/empty
   + C extractor/ex-generic-v2/without-rows
+  + C extractor/keboola.ex-aws-s3/om-default-bucket
   + C extractor/keboola.ex-db-mysql/with-rows
   + R extractor/keboola.ex-db-mysql/with-rows/rows/disabled
```

Comment:
V tomto commite pridavam do testu `template-create` config s default bucket.
Ako som uz pisal skor, default bucket mapper este nefunguje s template (je potrebne ho upravit).

No zatial to funguje aspon s nahradzovanim config/row ID v stringu.